### PR TITLE
Task #103849: Data deny set field in reports

### PR DESCRIPTION
**Pull request by amit4106udale**

### CODE REVIEW

File: tjreports/administrator/sql/install.mysql.utf8.sql [hunk 1]
```diff
@@ -7,6 +7,7 @@ CREATE TABLE IF NOT EXISTS `#__tj_reports` (
   `parent` int(11) NOT NULL,
   `default` tinyint(4) NOT NULL,
   `userid` int(11) NOT NULL,
+  `datadenyset` int(11),
   `param` text NOT NULL,
   PRIMARY KEY (`id`)
 ) ENGINE=InnoDB  DEFAULT CHARSET=latin1 AUTO_INCREMENT=57 ;
```
Comment (twsvaishali) [inline tjreports/administrator/sql/install.mysql.utf8.sql]: DO not provide engine

File: tjreports/languages/administrator/en-GB/en-GB.com_tjreports.ini [hunk 2]
```diff
@@ -87,13 +87,23 @@ COM_TJREPORTS_FORM_DESC_REPORT_DEFAULT="Select Yes, If you want to make this rep
 COM_TJREPORTS_YES="Yes"
 COM_TJREPORTS_NO="No"
 COM_TJREPORTS_FORM_LBL_REPORT_PERMISSIONS="Permissions"
+COM_TJREPORTS_FORM_LBL_REPORT_DATA_DENY_SET = "Data Deny set"
```
Comment (twsvaishali) [inline tjreports/languages/administrator/en-GB/en-GB.com_tjreports.ini]: Do not add space before or after =, in language file

File: tjreports/administrator/sql/install.mysql.utf8.sql (hunk 1 above)
Comment (twsvaishali) [inline tjreports/administrator/sql/install.mysql.utf8.sql]: @hemantpurohit1  DO not provide engine

File: tjreports/languages/administrator/en-GB/en-GB.com_tjreports.ini (hunk 2 above)
Comment (twsvaishali) [inline tjreports/languages/administrator/en-GB/en-GB.com_tjreports.ini]: @hemantpurohit1  Do not add space before or after =, in language file

File: tjreports/administrator/sql/install.mysql.utf8.sql (hunk 1 above)
Comment (amit4106udale) [inline tjreports/administrator/sql/install.mysql.utf8.sql]: @twsvaishali - resolved

File: tjreports/languages/administrator/en-GB/en-GB.com_tjreports.ini (hunk 2 above)
Comment (amit4106udale) [inline tjreports/languages/administrator/en-GB/en-GB.com_tjreports.ini]: @twsvaishali - done

File: tjreports/administrator/sql/install.mysql.utf8.sql [hunk 3]
```diff
@@ -10,4 +10,4 @@ CREATE TABLE IF NOT EXISTS `#__tj_reports` (
   `datadenyset` int(11),
   `param` text NOT NULL,
   PRIMARY KEY (`id`)
-) ENGINE=InnoDB  DEFAULT CHARSET=latin1 AUTO_INCREMENT=57 ;
+) DEFAULT CHARSET=latin1 AUTO_INCREMENT=57 ;
```
Comment (twsvaishali) [inline tjreports/administrator/sql/install.mysql.utf8.sql]: autoincremnet 1 pahije na

Comment (coolbung) [inline tjreports/administrator/sql/install.mysql.utf8.sql]: Remove default charset as well.

**Pull request merged**